video_xblock/static/css/videojs.css

```diff
@@ -451,8 +451,8 @@ body {
     font-family: "Open Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
 }
 .vjs-playback-rate .vjs-menu .vjs-menu-content li:focus {
-    color: #0ea6ec;
-    border-left: 2px solid #0ea6ec;
+    color: #cb598d;
```

Comment:
I'm not sure that it's a good idea to have a static color code. Could we inherit the theme color?

Reply:
We can't move theme color to xblock in simple way. We use neutral color on this dropdown, which is using on other player elements

Reply:
Got it. Thanks.